### PR TITLE
Visibility checkboxes does not match preview #4509

### LINKED ISSUE #4509: Visibility checkboxes does not match preview
Here's the problematic question:
image
image
Here's another question in the same session but seem to work properly:
image
image

### PR DESCRIPTION
**Pull request by kanghj**
Fixes #4509

@LowWeiLin for your review. If you are not free today, you can assign someone else.

### CODE REVIEW

**Comment by LowWeiLin** [comment]
Looks good, but are we handle this on the back-end by ignoring the irrelevant checkboxes?

**Comment by LowWeiLin** [comment]
I think the real issue is in the function removeIrrelevantVisibilityOptions()?

**Comment by kanghj** [comment]
Changes made

File: src/main/webapp/js/instructorFeedbackEdit.js [hunk 1]
```diff
@@ -413,7 +413,10 @@ function tallyCheckboxes(qnNumber) {
     for (var checkboxType in checkboxTypes) {
         var checked = [];
         $(checkboxType + qnNumber + ':checked').each(function() {
-            checked.push($(this).val());
+            // add checkbox value if the checkbox is not hidden
+            if ($(this).closest('tr, hide').css('display') !== 'none') {
+                checked.push($(this).val());
+            }
```
Comment (LowWeiLin) [inline src/main/webapp/js/instructorFeedbackEdit.js]: No need for the js part anymore right?

**Comment by LowWeiLin** [comment]
Can merge

**Comment by LowWeiLin** [comment]
@damithc  Merged this fix into release branch, V5.58.01

**Pull request merged**